spellbook/tests/nomad/ethereum/nomad_ethereum_assert.sql

```diff
@@ -0,0 +1,11 @@
+-- Bootstrapped correctness test against legacy Postgres values.
```

Comment:
is this comment applicable or leftover?

Reply:
I copied a file from others sample. Removed.